### ISSUE TITLE
Reproduce the result of localnet

### ISSUE THREAD

**Issue by s7ev3n** [wvangansbeke/Sparse-Depth-Completion#3]
Hi,
Thanks and waiting for your code release.

I am trying to reproduce the result of localnet. The localnet takes `1216x256x1` velodyne_raw(not normalized,but divide 256 to obtain real depth value) as input, and a simple conv layer with `stride=2, kernel_size=3, out_channels=32`, following two hourglasses as in the paper, then a tranconv layer of `stride=2, kernel_size=3, out_channels=32` and a `1x1` conv layer with `out_channels=2`, the confidence map after softmax adds on the local depth prediction.
I trained on kitti depth dataset with adam and `lr=0.001` for 40 epochs and 48 batchsize, the loss is masked mae loss. However, my result is much worse, the best rmse is around 1800 compared with your localnet result 995.

Is there any insight to improve the localnet? Thanks so much :)

-s7ev3n

**Comment by wvangansbeke** [comment]
Hi @s7ev3n,

Thank you for waiting. I updated the code in this repo. Feel free to take a look. It's easier to spot the differences this way.

Kind regards,
Wouter
